DigitalLearningSolutions.Data/Services/CustomPromptsService.cs

```diff
@@ -4,7 +4,7 @@
 
     public interface ICustomPromptsService { }
 
-    public class CustomPromptsService
+    public class CustomPromptsService : ICustomPromptsService
```

Comment:
Do we still need the interface? Can we not now make this static instead. (It should perhaps not be named CustomPromptService too as it's now become more of a helper class

Reply:
Changed to a helper class called CustomPromptHelper and renamed the helper class in the Web project to CentreCustomPromptHelper to avoid confusion